pipeline/builder/builder_test.go

```diff
@@ -446,14 +447,51 @@ func TestBuilderPipelineStages(t *testing.T) {
 		builder := builder.New(pipeline)
 		spinnaker, err := builder.Pipeline()
 		require.NoError(t, err, "error generating pipeline json")
-		var d v1.Deployment
+		var d appsv1.Deployment
 		u, _ := runtime.DefaultUnstructuredConverter.ToUnstructured(spinnaker.Stages[0].(*types.ManifestStage).Manifests[0])
 		err = runtime.DefaultUnstructuredConverter.FromUnstructured(u, &d)
 		require.NoError(t, err)
 		container := d.Spec.Template.Spec.Containers[0]
 		assert.Equal(t, "300Mi", container.Resources.Limits.Memory().String())
 		assert.Equal(t, "400m", container.Resources.Limits.Cpu().String())
 	})
+	t.Run("Overrides container overrides only cpu with resources set in CronJob", func(t *testing.T) {
```

Comment:
this test name makes is seem like you're only looking to override the cpu resource definitions. but the assertions seem like you're injecting in / overriding both the requests and the limits. can you update the test name please?